src/index.ts

```diff
@@ -69,7 +69,7 @@ async function getPkgLicense(pkg: PkgInfo): Promise<LicenseInfo> {
   const url = new URL(REGISTRY);
   url.pathname = pkg.name;
   // Get registry info
-  await new Promise<void>((resolve) => {
+  await new Promise<boolean>((resolve) => {
```

Comment:
Would it work if you just remove this Promise constructor and instead just `return await superagent.get()...`?  

That way you wouldn't have to manually call resolve() everywhere.

Reply:
Maybe. I'm not familiar with superagent, so I didn't do that.